### PR TITLE
convert \n into <br> in ObjectFieldValue

### PR DESCRIPTION
**Pull request by wipascal**

### CODE REVIEW

File: bread/layout/base.py [hunk 1]
```diff
@@ -137,6 +138,8 @@ def resolve(self, context):
             value = self.formatter(value)
         if isinstance(value, models.Manager):
             value = ", ".join([str(x) for x in value.all()])
+        if isinstance(value, str):
+            value = linebreaksbr(value, autoescape=False)
```
Comment (wipascal) [inline bread/layout/base.py:142]: Actually `autoescape=True` seems to lead to exactly the same result... Not sure which one makes more sense.

Comment (saemideluxe) [reply bread/layout/base.py:142]: Okay, then we can leave it with the default I guess.

**Review by saemideluxe — APPROVED** [review]

**Comment by wipascal** [comment]
sorry I just committed another small thing here. should have made another branch

**Review by saemideluxe — APPROVED** [review]
Sure

**Pull request merged**